### DISCUSSION TITLE
sample limit exceeded event is not logged in 2.39.1

### DISCUSSION THREAD

**Discussion by timchenko-a** [prometheus/prometheus#11636] [Q&A]
Hi. We recently updated prometheus 2.36.2 -> 2.39.1 and after update I noticed `sample limit exceeded` events are not logged any more. Before those events were logged with warn level. So right now the only way to find the failing target is via /targets endpoints. I wasn't able to find anything related in changelogs, so I wonder if this could be a bug or it's rather an expected behaviour?

**Comment by roidelapluie** [comment]
I created an issue out of this:

https://github.com/prometheus/prometheus/issues/11736

**Comment by roidelapluie** [comment]
This was a bug that was fixed in #9996. Note that I could not reproduce the logging in 2.36.2.
